IDBOOKAPI/api/viewsets.py

```
@@ -1107,15 +1338,20 @@ def post(self, request):
 
 class BasicAuthKeyViewSet(APIView):
     def get(self, request):
```

Ruff (0.14.8)

Unused method argument: `request`

(ARG002)